sky/job/dashboard/dashboard.py

```diff
@@ -53,11 +53,12 @@ def home():
     rows = managed_jobs.format_job_table(all_managed_jobs,
                                          show_all=True,
                                          return_rows=True)
+    rows = [[''] + row for row in rows]
```

Comment:
Add a comment on what the 1st column is.

Reply:
Added. Thanks!